_docs_use-cases/use-case-consumption-delete-a-repositoryattribute.md

```diff
@@ -0,0 +1,60 @@
+---
+# Start automatic generation
+permalink: use-case-consumption-delete-a-repositoryattribute
+published: true
+title: "Delete a RepositoryAttribute"
+type: use-case
+toc: true
+sidebar:
+  - title: "Integrate Enmeshed"
+    nav: "docs_integrate"
+properties:
+  - id: RA7
+  - component: Runtime
+  - layer: Consumption
+  - facade: AttributesFacade
+  - function: deleteRepositoryAttribute
+  - description:
+  - feature category: Normalized attributes
+  - tech category: Attributes
+  - status: DONE
+  - documentation status:
+  - comments:
+  - actor: Identity
+  - trigger: REST API
+  - precondition:
+  - result:
+  - priority: n/a
+  - complexity: n/a
+  - size: n/a
+  - created_at:
+  - changed_at:
+  - api_route_regex:
+  - published: default
+  - link: use-case-consumption-delete-a-repositoryattribute
+require:
+required_by:
+# End automatic generation
+---
+
+{{properties.description}}
+
+{% include properties_list.html %}
+
+This use-case allows you to delete a RepositoryAttribute, i.e. an own [LocalAttribute]({%link _docs_integrate/data-model-overview.md %}#localattribute) without `shareInfo`.
```

Comment:
I'd reformulate: "This use-case allows you to delete a RepositoryAttribute, which is a [LocalAttribute]({%link _docs_integrate/data-model-overview.md %}#localattribute) that is owned by yourself and whose `shareInfo` property is undefined."